test/integration/actions-test.js

```diff
@@ -4,162 +4,155 @@
 /* globals it */
 
 const {server, chai, mockAdapter} = require('../common');
+const {
+  TEST_USER,
+  createUser,
+  headerAuth,
+} = require('../user');
 
-var Constants = require('../../constants');
+const pFinal = require('../promise-final');
+
+const Constants = require('../../constants');
 const WebSocket = require('ws');
 
-it('removes all existing actions', async () => {
+describe('actions/', function() {
+  let jwt;
 
-  let res = await chai.request(server)
-    .get(Constants.ACTIONS_PATH)
-  res.should.have.status(200);
-  res.body.should.be.a('array');
+  beforeEach(async () => {
+    jwt = await createUser(server, TEST_USER);
+  })
 
-  // Issue a request to delete all current actions
-  let deleteReqs = res.body.map(action => {
-    return chai.request(server)
-      .delete(Constants.ACTIONS_PATH + '/' + action.id);
+  it('GET with no actions', async () => {
+    const res = await chai.request(server)
+      .get(Constants.ACTIONS_PATH)
+      .set(...headerAuth(jwt));
+    res.should.have.status(200);
+    res.body.should.be.a('array');
+    res.body.length.should.be.eql(0);
   });
 
-  const responses = await Promise.all(deleteReqs);
-  for (let res of responses) {
-    res.should.have.status(204);
-  }
-
-  res = await chai.request(server)
-      .get(Constants.ACTIONS_PATH);
-
-  res.body.should.be.a('array');
-  res.body.length.should.be.eql(0);
-});
-
-it('GET with no actions', async () => {
-  const res = await chai.request(server)
-    .get(Constants.ACTIONS_PATH);
-  res.should.have.status(200);
-  res.body.should.be.a('array');
-  res.body.length.should.be.eql(0);
-});
-
-it('should fail to create a new action (empty body)', async () => {
-  try {
-    await chai.request(server)
+  it('should fail to create a new action (empty body)', async () => {
+    const err = await pFinal(chai.request(server)
```

Comment:
@hobinjk This is an alternative style to the try/catch (I like this better because it's less boilerplate and no gotchas). I chose `promise-final` (referencing "finally") as it will return either the error OR the result (without throwing).